crates/launcher/src/main.rs

```diff
@@ -54,14 +54,16 @@ fn run() -> LauncherResult<()> {
 
     // TODO: merge
     if let Some(mut profile) = profiles.into_iter().next() {
-        let ordered_natives = sort_dependencies(profile.natives())
```

Comment:
Can you share the config this code is causing problems with?

Reply:
```toml
profileVersion = "v1"

[[natives]]
path = "waygate_client.dll"
optional = true

[[natives]]
path = "eldenring_alt_saves.dll"
optional = true

# [[packages]]
# id = "test-mod-emevd-rave"
# source = "Z:\\home\\vincent\\me3-test-mod\\emevd-rave"
#
# [[packages]]
# id = "test-mod-clevers"
# source = "Z:\\home\\vincent\\me3-test-mod\\clevers"

[[packages]]
id = "err"
source = "Z:\\home\\vincent\\me3-test-mod\\err"
```